app/views/spree/admin/orders/index.html.haml

```diff
@@ -24,15 +24,47 @@
       %th
         = t(:products_distributor)
       - if @show_only_completed
-        %th= sort_link @search, :completed_at, t(:completed_at, :scope => 'activerecord.attributes.spree/order')
+        %th
+          %a{'ng-click' => "sortOptions.toggle('completed_at')"}
+            = t(:completed_at, scope: 'activerecord.attributes.spree/order')
+            %span{'ng-show' => "sorting == 'completed_at asc'"}= "&#x25B2;".html_safe
+            %span{'ng-show' => "sorting == 'completed_at desc' || sorting === undefined"}= "&#x25BC;".html_safe
       - else
-        %th= sort_link @search, :created_at,   t(:created_at, :scope => 'activerecord.attributes.spree/order')
-      %th= sort_link @search, :number,         t(:number, :scope => 'activerecord.attributes.spree/order')
-      %th= sort_link @search, :state,          t(:state, :scope => 'activerecord.attributes.spree/order')
-      %th= sort_link @search, :payment_state,  t(:payment_state, :scope => 'activerecord.attributes.spree/order')
-      %th= sort_link @search, :shipment_state, t(:shipment_state, :scope => 'activerecord.attributes.spree/order')
-      %th= sort_link @search, :email,          t(:email, :scope => 'activerecord.attributes.spree/order')
-      %th= sort_link @search, :total,          t(:total, :scope => 'activerecord.attributes.spree/order')
+        %th
+          %a{'ng-click' => "sortOptions.toggle('created_at')"}
+            = t(:created_at, scope: 'activerecord.attributes.spree/order')
+            %span{'ng-show' => "sorting == 'created_at asc'"}= "&#x25B2;".html_safe
+            %span{'ng-show' => "sorting == 'created_at desc'"}= "&#x25BC;".html_safe
+      %th
```

Comment:
There is a lot of code duplication here. Maybe a partial with local variables would be good?